library/ssl_tls12_server.c

```diff
@@ -2200,8 +2254,6 @@ static int ssl_write_server_hello(mbedtls_ssl_context *ssl)
      *     0  .   0   handshake type
      *     1  .   3   handshake length
      *     4  .   5   protocol version
-     *     6  .   9   UNIX time()
-     *    10  .  37   random bytes
```

Comment:
I do not think so. 

I prefer copy the structure from RFC directly and remove unnecessary part like ln 2268 to 2284. I did not change origin lines just because it is out of scope.

Reply:
I just replace the comments